src/index.js

```diff
@@ -109,7 +110,10 @@ class RBSheet extends Component {
         visible={modalVisible}
         supportedOrientations={SUPPORTED_ORIENTATIONS}
         onRequestClose={() => {
-          if (closeOnPressBack) this.setModalVisible(false);
+          if (closeOnPressBack) {
```

Comment:
The prop closeOnPressBack, is a boolean prop, by default is true and determine if the bottom sheet will close or not when the user makes click outside the bottom sheet.